### PR TITLE
Illegal gen 1 movesets

### PR DESCRIPTION
**Pull request by VeteranPadgett**
We don't keep the level at which moves are learned, so I just had to list all the combinations. Also, I'm not sure why RBY Ubers is in Stadium w/ Tradebacks instead of Yellow gen. If it's supposed to have tradebacks then I can just add a commit to remove RBY Ubers from the check.

### CODE REVIEW

File: scripts/tierchecks.js [hunk 1]
```diff
@@ -122,6 +122,33 @@ tier_checker.add_new_check(INCLUDING, ["BW2 NU"], function evioliteCheck(src, te
     }
 });
 
+tier_checker.add_new_check(INCLUDING, ["Red/Blue", "Yellow", "Stadium", "RBY Ubers", "RBY OU", "RBY BL", "RBY UU", "RBY NU", "RBY LC"], function noTradebacksCheck(src, team, tier) {
+    var ret = [];
+    var illegal_combinations = {"Kakuna": {"Harden": ["Poison Sting", "String Shot"]}, "Beedrill": {"Harden": ["Poison Sting", "String Shot"]},  "Fearow": {"Payday": ["Peck", "Mirror Move", "Drill Peck"]}, "Nidoking": {"Thrash": ["Fury Attack"]}, "Rapidash": {"Payday": ["Tail Whip", "Growl"]}, "Exeggutor": {"Stomp": ["Sleep Powder", "Stun Spore", "Poison Powder"]}, "Jolteon": {"Focus Energy": ["Thundershock"]}, "Flareon": {"Focus Energy": ["Ember"]}};
```
Comment (Fuzzysqurl) [inline scripts/tierchecks.js]: If we do end up fixing it via this route, Fearow should only be restricted on Peck, as that is the only 100% confirmed one for it to be locked out of. magic9mushroom brought up a good point regarding this that makes sense (The Fearow is at level 20, not 40, which still allows it to get Drill Peck and Mirror Move)

**Comment by VeteranPadgett** [comment]
Done

**Comment by CrystalMoogle** [comment]
Is there really no way to do it in the teambuilder? :(

**Comment by VeteranPadgett** [comment]
I wanted to base it off of the tiers, but I could do it in teambuilder and base it off of the subgen.

**Pull request closed without merging**